packages/cli/src/commands/dev/dev.ts

```diff
@@ -104,6 +106,13 @@ export default async function dev(
     devCommand = process.env.VERCEL_DEV_COMMAND;
   }
 
+  if (!devCommand) {
+    // If there is no Development Command, we must delete the
+    // v3 Build Output because it will incorrectly be detected by
+    // `@vercel/static-build` when it executes BuildOutputV3.getBuildOutputDirectory()
+    await fs.remove(OUTPUT_DIR);
```

Comment:
Can we log out that this directory is being removed? It may save a user from some confusion.

Reply:
Add in 9587966 👍